Sources/App/Slack/Slack.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  Slack.swift
+//  App
+//
+//  Created by Allen Humphreys on 3/2/19.
+//
+
+import SlackKit
+import Vapor
+
+struct Slack: ServiceType {
+    private let slackKit = SlackKit()
+    private let listener: SlackListener
```

Comment:
I'm not sure how this all pieces together, but I'm worried about possible cyclical references with some of these

Reply:
cyclical references? Like retain cycles? Yes there is definitely circular references but all the instances are long lived